lib/ccflay.rb

```diff
@@ -84,3 +84,25 @@ def pure_ruby_hash # :nodoc: see above
     hash
   end
 end
+
+class Sexp
```

Comment:
Yes, 1) it's my style. 2) the previous block needs to be pushed back to flay itself as it is a measurable win for everyone. It's even faster than ruby's default hash.